examples/platforms/cc2538/random.c

```diff
@@ -66,3 +69,32 @@ uint32_t otPlatRandomGet(void)
 
     return mlcg;
 }
+
+ThreadError otPlatSecureRandomGet(uint16_t aInputLength, uint8_t *aOutput, uint16_t *aOutputLength)
+{
+    uint16_t length = 0;
+    uint32_t random;
+
+    while (length < aInputLength)
+    {
+        random = otPlatRandomGet();
+
+        if (aInputLength > (sizeof(random) + length))
```

Comment:
This seems more complex that is necessary.  How about a simple for-loop that just call `otPlatRandomGet()` for each byte?  This is example code never to be used in production anyway.